inspire_dojson/hep/rules/bdFFT.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""DoJSON rules for FFT."""
+
+from __future__ import absolute_import, division, print_function
+
+from datetime import datetime
+
+from dojson import utils
+
+from ..model import hep, hep2marc
+from ...utils import absolute_url, force_single_element
+from ...utils.helpers import force_list, maybe_int
+
+
+@hep.over('_fft', '^FFT[^%][^%]')
+@utils.for_each_value
+def _fft(self, key, value):
+    def _get_creation_datetime(value):
+        dt = datetime.strptime(value.get('s'), '%Y-%m-%d %H:%M:%S')
+        return dt.isoformat()
+
+    def _is_context(value):
+        return value.get('f', '').endswith('context')
+
+    if _is_context(value):
+        return
+
+    return {
+        'creation_datetime': _get_creation_datetime(value),
+        'description': value.get('d'),
+        'filename': value.get('n'),
+        'flags': force_list(value.get('o')),
+        'format': value.get('f'),
+        'path': value.get('a').split(';')[0],
+        'status': value.get('z'),
+        'type': value.get('t'),
+        'version': maybe_int(value.get('v')),
+    }
+
+
+@hep2marc.over('FFT', '^documents')
+@utils.for_each_value
+def documents2marc(self, key, value):
+    return {
+        'd': value.get('description'),
+        'a': absolute_url(value.get('url')),
+        't': 'Main',
```

Comment:
Mmh. Here there should be `Elsevier` etc. Is this information lost in translation?